libs/ledger-live-common/src/deposit/helper.ts

```diff
@@ -11,7 +11,7 @@ const groupCurrenciesByProvider = (
   }
   const assetsByProviderId: Record<string, CurrenciesByProviderId> = {};
   for (const ledgerCurrency of currenciesSupported) {
-    const asset = assetsByLedgerId[ledgerCurrency.id];
+    const asset = assetsByLedgerId[ledgerCurrency.id.toLowerCase()];
     if (asset) {
       if (!assetsByProviderId[asset.providerId]) {
         assetsByProviderId[asset.providerId] = {
```

Comment:
even there maybe also?

Reply:
@KVNLS @mcayuelas-ledger @cgrellard-ledger things coming out of `getMappedAssets()` are properly formatted (i.e lowercased) and coming from our APIs. It's unnecessary to lowercase it, we could do it for the sake of it, but I think it would be a stretch. That being said I can do it if you want to but I wouldn't do it imho.

Reply:
At least we will be resilient for changes, if they decide to upperCase tomorrow.

Reply:
Done @KVNLS